webapp/src/Command/ScoreboardMergeCommand.php

```diff
@@ -59,7 +59,10 @@ public function __construct(
 
     protected function configure(): void
     {
-        $this
+        $this->setName('scoreboard:merge')
```

Comment:
This is not needed anymore because of line 40

Reply:
right; removed